exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -156,7 +157,16 @@ private Adapter() {}
 
     // name is a top-level property in OpenTelemetry
     builder.addFields(
-        Model.KeyValue.newBuilder().setKey(KEY_LOG_MESSAGE).setVStr(event.getName()).build());
+        Model.KeyValue.newBuilder().setKey(KEY_LOG_EVENT).setVStr(event.getName()).build());
+
+    int droppedAttributesCount = event.getTotalAttributeCount() - event.getAttributes().size();
```

Comment:
I'd be in favor of adding a method to the Event which does this work for you.

Reply:
👍  What about adding a default method？
```
default boolean getDroppedAttributesCount() {
      return getTotalAttributeCount() == getAttributes().size();
    }
```